### PR TITLE
frontend: added logout functionality

### PR DESCRIPTION
**Pull request by IvelinYR**
-added logout functionality, when the user logouts, the cookie is deleted
-added error messages
-naming improvements

### CODE REVIEW

File: frontend/src/reducers/account.js [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+                error: action.error.response.data.Message
+            });
         case 'LOAD_ACCOUNTS_SUCCESS':
-            let accountList = action.payload.data
-            return accountList;
+            let accountsList = action.payload.data;
+            return accountsList;
+        case 'LOGOUT_SUCCESS':
+            const nameCookie = "SID";
+            Cookies.remove(nameCookie);
```
Comment (mgenov) [inline frontend/src/reducers/account.js]: You can just call Cookies.remove("SID") 

It's strange that login is not handled here, if it's here, then the name of the cookie could be extracted as constant.

**Review by mgenov — CHANGES_REQUESTED** [review]

File: frontend/src/reducers/account.js (hunk 1 above)
Comment (IvelinYR) [reply frontend/src/reducers/account.js]: I will put them in one  reducer and I will change cookie.

**Comment by IvelinYR** [comment]
Ready for review.

**Review by mgenov — APPROVED** [review]
LGTM ! 

Keep in mind that the reducer with usages of navigation history will be hard for testing.

**Pull request merged**